### PR TITLE
Blog post about OIDC certifications

### PR DESCRIPTION
**Pull request by mposolda**
@stianst @tnorimat @andymunro Do you please have a chance to review this? It is draft for the new blog post about OIDC certifications.

### CODE REVIEW

File: blog/2022/oidc-certifications.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+See the https://openid.net/certification/#OPs[OpenID Connect certifications page] for the details.
+
+- Keycloak 18.0.0 is certified as OpenID Connect Logout Provider with all logout profiles (RP-Initiated OP, Session OP, Front-Channel OP, Backchannel OP).
+See the https://openid.net/certification/#OPs[OpenID Connect certifications page (logout section)] for the details.
+
+- Keycloak 15.0.2 is certified as https://consumerdatastandardsaustralia.github.io/standards/#security-profile[Australia CDR], which is the extension based on existing
+FAPI 1 Advanced Final certification, which Keycloak already obtained before. See the https://openid.net/certification/#FAPI_OPs[FAPI certifications page] for the details.
+
+This milestone was achieved due the hard work of the awesome Keycloak community, who contributed lots of features related to OpenID Connect Protocol, OpenID Connect Logout and FAPI.
+The special Thanks go to the https://github.com/keycloak/kc-sig-fapi/blob/main/members.adoc[FAPI WG], who helped a lot with the FAPI and OpenID Connect related features and especially
```
Comment (tnorimat) [inline blog/2022/oidc-certifications.adoc:21]: `FAPI WG` might be `FAPI-SIG`.

**Review by tnorimat — COMMENTED** [review]
@mposolda I added one comment. Could you check it?

File: blog/2022/oidc-certifications.adoc (hunk 1 above)
Comment (mposolda) [reply blog/2022/oidc-certifications.adoc]: @tnorimat Fixed. Thanks!

**Review by tnorimat — APPROVED** [review]

**Comment by tnorimat** [comment]
@mposolda LGTM.

**Pull request merged**